packages/picker-button/src/PickerButton.ts

```diff
@@ -57,6 +60,7 @@ export class PickerButton extends SizedMixin(
             root: true,
             uiicononly: !this.hasText,
             textuiicon: this.hasText,
+            'spectrum-PickerButton--quiet': this.quiet,
```

Comment:
If you update https://github.com/adobe/spectrum-web-components/blob/main/packages/picker-button/src/spectrum-config.js#L212-L215 to be `spectrum-PickerButton--quiet` instead of `spectrum-PickerButton--low` we won't need to effect the render method with this change.